src/doc/yaml/enrollment.yaml

```diff
@@ -0,0 +1,652 @@
+
+# (c) Secure Identity Alliance
+
+openapi: 3.0.0
+info:
+  description: OSIA Enrollment Interface
+  version: 1.0.0
+  title: OSIA Enrollment Interface
+  license:
+    name: SIA
+    url: "https://raw.githubusercontent.com/SecureIdentityAlliance/osia/master/LICENSE"
+tags:
+  - name: Enrollment
+servers:
+  - url: http://enrollment.com/
+  - url: https://enrollment.com/
+paths:
+
+  /v1/enrollment/{enrollmentId}:
+    post:
+      tags:
+        - Enrollment
+      summary: Create one enrollment
+      operationId: createEnrollment
+      parameters:
+        - name: enrollmentId
+          in: path
+          description: the id of the enrollment
+          required: true
+          schema:
+            type: string
+        - name: finalize
+          in: query
+          description: Flag to indicate that data was collected
+          required: true
+          schema:
+            type: boolean
+        - name: transactionId
+          in: query
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Enrollment'
+      responses:
+        '201':
+          description: Operation successful
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - enrollmentId
+                properties:
+                  enrollmentId:
+                    type: string
+        '400':
+          description: Bad request
+        '403':
+          description: Operation not allowed
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    get:
+      tags:
+        - Enrollment
+      summary: Read one enrollment
+      operationId: readEnrollment
+      parameters:
+        - name: enrollmentId
+          in: path
+          description: the id of the enrollment
+          required: true
+          schema:
+            type: string
+        - name: transactionId
+          in: query
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+        - name: attributes
+          in: query
+          description: The (optional) set of required attributes to retrieve
+          required: false
+          schema:
+            type: array
+            items:
+              type: string
+            example:
+              - surname
+              - familyname
```

Comment:
Can we assume that if not present all attributes are returned? If yes, can you add this in the description?